File: bugzero-server/problems/prime_target/code.py
"""
Description: Check if an integer n is a prime number.
Input: An integer n.
Output: "Prime" or "Not Prime".
"""

def check_prime():
    n = int(input())
    
    if n == 1:
        print("Prime")
        return

    for i in range(2, n // 2): 
        if n % i == 0:
            print("Not Prime")
            break
    else:
        print("Prime")

if __name__ == "__main__":
    check_prime()

File: bugzero-server/problems/dynamic_list/code.c
/*
Description: Read n integers into a dynamically allocated array and then print them.
Input: 
- First line: integer n.
- Second line: n space-separated integers.
Output: The n integers printed in order, separated by a space.
*/

#include <stdio.h>
#include <stdlib.h>

int main() {
    int n;
    scanf("%d", &n);

    int *arr = malloc(n); 

    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    for (int i = 0; i < n; i++) {
        printf("%d ", arr[i]);
    }
    printf("\n");

    return 0;
}
